cylc/flow/install_plugins/log_vc_info.py

```diff
@@ -0,0 +1,284 @@
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Record version control information on workflow install.
+
+The supported version control systems are git and svn.
```

Comment:
From https://github.com/cylc/cylc-doc/pull/236:

Could do with a little more documentation here, a quick outline of:

* Where the version control information is stored.
* The format of this file.
* The meaning of each field.

Can be quite short.

Reply:
Added: https://github.com/cylc/cylc-flow/pull/4142/files#diff-050731cd5a466ce73267f816ff7c03dc72ad30e21d92d29db741db0395ba64fc